### DISCUSSION TITLE
"deckgl = TypeError: deck.DeckGL is not a constructor" in previously published Observable Notebook

### DISCUSSION THREAD

**Discussion by cornhundred** [visgl/deck.gl#7739] [Q&A]
Hi, I'm seeing this error 

`deckgl = TypeError: deck.DeckGL is not a constructor`

when running this cell

```
deckgl = {
  return new deck.DeckGL({
    container,
    controller: true,
    initialViewState: {longitude: 0, latitude: 15, zoom: 1}
  });
}
```

in this [ObservableHQ notebook](https://observablehq.com/@pessimistress/deck-gl-tutorial-subclassing-a-layer)

deck.gl is being imported like this

```
deck = require.alias({
  // optional dependencies
  h3: {}
})('deck.gl@latest/dist.min.js')
```

and the outputs of the cell are

```
deck = ƒ(O, M)
```

so it looks like deck.gl might not be getting imported correctly using this approach now (I don't see a version also).

**Comment by Pessimistress** [comment]
Fixed in 8.9.1
